src/client/index.js

```diff
@@ -257,6 +266,87 @@ export class BncClient {
     return this._broadcastDelegate(signedTx)
   }
 
+  /**
+   * Create and sign a multi send tx
+   * @param {String} fromAddress
+   * @param {Array} transfers
+   * @example
+   * const transfers = [
+   * {
+   *   "to": "tbnb1p4kpnj5qz5spsaf0d2555h6ctngse0me5q57qe",
+   *   "coins": [{
+   *     "denom": "BNB",
+   *     "amount": 10
+   *   }]
+   * },
+   * {
+   *   "to": "tbnb1scjj8chhhp7lngdeflltzex22yaf9ep59ls4gk",
+   *   "coins": [{
+   *     "denom": "BTC",
+   *     "amount": 10
+   *   }]
+   * }]
+   * @param {String} memo optional memo
+   * @param {Number} sequence optional sequence
+   * @return {Promise} resolves with response (success or fail)
+   */
+  async multiSend(fromAddress, transfers, memo="", sequence=null) {
+    if(!fromAddress) {
+      throw new Error("fromAddress should not be falsy")
+    }
+
+    if(!Array.isArray(transfers)) {
+      throw new Error("transfers should be array")
+    }
+
+    checkTransfers(transfers)
+
+    transfers.forEach(item=>{
+      item.coins.forEach(coin=>{
+        coin.amount = Number(coin.amount * Math.pow(10, 8))
+      })
+    })
+
+    const fromAddrCode = crypto.decodeAddress(fromAddress)
+    const inputs = [{ address: fromAddrCode, coins: [] }]
+    const outputs = []
+
+    transfers.forEach((item)=>{
+      const toAddcCode = crypto.decodeAddress(item.to)
+      inputs[0].coins = inputs[0].coins.concat(item.coins)
+      outputs.push({address: toAddcCode, coins: item.coins})
+    })
+
+    const msg = {
+      inputs,
+      outputs,
+      msgType: "MsgSend"
+    }
+
+    const signInputs = [{ address: fromAddress, coins: [] }]
+    const signOutputs = []
+    transfers.forEach((item, index)=>{
+      signOutputs.push({ address: item.to, coins: [] })
+      item.coins.forEach(c=>{
+        const coin = {
+          denom: c.denom,
+          amount: c.amount
+        }
+
+        signInputs[0].coins.push(coin)
+        signOutputs[index].coins.push(coin)
```

Comment:
The coins should also be sorted by `denom`, example:

https://testnet-dex.binance.org/api/v1/tx/04740D6CA09DE8C42B8E27754AC5A75A3B5E2B7D728A885C7E5E7BE06EA3C56A?format=json